servers.md

```diff
@@ -47,6 +47,14 @@ server(...)
     ->identityFile();
 ~~~
 
+### 2-factor authentication with an identity file and password or google-authenticator:
+
+~~~ php
+server(...)
+    ->user('name')
+    ->identityFileAndPassword();
```

Comment:
I think you should replace by:
```
->identityFileAndPassword('/path/to/publicKey', '/path/to/privateKey', 'passPhrase', 'password');
```
And add note: Current this version only support PhpSecLib